.github/workflows/towerfulltest.yml

```diff
@@ -0,0 +1,27 @@
+name: Tower AWS full size tests
+# This workflow is triggered on published releases.
+# It can be additionally triggered manually with GitHub actions workflow dispatch button.
+# It runs the -profile 'test_full' on AWS batch via Tower
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+jobs:
+  run-tower:
+    name: Run AWS full tests
+    if: github.repository == 'Arcadia-Science/seqqc'
+    runs-on: ubuntu-latest
```

Comment:
should we pin these to a specific version?

Reply:
I don't feel super strongly about this. I kind of like that it will force us to keep things current on releases, even if that does cause some dependency issues. Those are things we should probably be aware of and address anyway so, 🤷‍♀️ . Happy to pin if you feel strongly about it

Reply:
Sounds good, this is fine